modules/filecheck/README.md

```diff
@@ -0,0 +1,104 @@
+<!--
+title: "Files and directories monitoring with Netdata"
+custom_edit_url: https://github.com/netdata/go.d.plugin/edit/master/modules/fileheck/README.md
+sidebar_label: "Files and Dirs"
+-->
+
+# filecheck
+
+This module monitors files and directories.
+
+File metrics:
+
+-   existence
+-   time since the last modification
+-   size
+
+Directory metrics:
+
+-   existence
+-   time since the last modification
+-   number of files
+
+## Charts
+
+Files and directories have their own set of charts.
+
+### Files
+
+-   File Existence in `boolean`
+-   File Time Since the Last Modification in `seconds`
+-   File Size in `bytes`
+
+### Directories
+
+-   Dir Existence in `boolean`
+-   Dir Time Since the Last Modification in `seconds`
+-   Dir Number of Files in `files`
+
+## Configuration
+
+Edit the `go.d/filecheck.conf` configuration file using `edit-config` from the Agent's [config
+directory](/docs/step-by-step/step-04.md#find-your-netdataconf-file), which is typically at `/etc/netdata`.
+
+```bash
+cd /etc/netdata # Replace this path with your Netdata config directory
+sudo ./edit-config go.d/filecheck.conf
+```
+
+Needs only path to a file or a directory. **The path doesn't support any wildcards**.
+
+Here is an example with the three jobs:
+
+```yaml
+jobs:
+ - name: files_dirs_example
+   files:
+     include:
+       - '/path/to/file1'
+       - '/path/to/file2'
+   dirs:
+     include:
+       - '/path/to/dir1'
+       - '/path/to/dir2'
+
+ - name: files_example
+   files:
+     include:
+       - '/path/to/file1'
+       - '/path/to/file2'
+
+ - name: dirs_example
+   dirs:
+     include:
+       - '/path/to/dir1'
+       - '/path/to/dir2'
+```
+
+For all available options, see the Filecheck collector's [configuration
+file](https://github.com/netdata/go.d.plugin/blob/master/config/go.d/filecheck.conf).
+
+## Limitations
+
+-   fil/dir path pattern doesn't support any wildcards
```

Comment:
I think there is a typo problem here, instead `fil/dir` I imagine the correct would be `file/dir`, because until now I could not see any reference to `fil`.

Reply:
There is invisible `e` 👻